Documentation/contributing.rst

```diff
@@ -812,303 +960,370 @@ Have a question about how the tests work or want to chat more about improving th
+
+   +-----------------------------------+---------------------------------------------------------------------------+
+   | Labels                            | When to set                                                               |
+   +===================================+===========================================================================+
+   | ``dont-merge/needs-release-note`` | Do NOT merge PR, needs a release note                                     |
+   +-----------------------------------+---------------------------------------------------------------------------+
+   | ``release-note/bug``              | This is a non-trivial bugfix                                              |
+   +-----------------------------------+---------------------------------------------------------------------------+
+   | ``release-note/major``            | This is a major feature addition                                          |
+   +-----------------------------------+---------------------------------------------------------------------------+
+   | ``release-note/minor``            | This is a minor feature addition                                          |
+   +-----------------------------------+---------------------------------------------------------------------------+
+
+#. Check for upgrade compatibility impact and if in doubt, set the label
+   ``upgrade-impact`` and discuss in the Slack channel.
+
+   +--------------------------+---------------------------------------------------------------------------+
+   | Labels                   | When to set                                                               |
+   +==========================+===========================================================================+
+   | ``upgrade-impact``       | The code changes have a potential upgrade impact                          |
+   +--------------------------+---------------------------------------------------------------------------+
+
+#. When everything looks OK, approve the changes.
+
+#. When all review objectives for all ``CODEOWNERS`` are met and all CI tests
+   have passed, you may set the ``ready-to-merge`` label to indicate that all
+   criteria have been met.
+
+   +--------------------------+---------------------------------------------------------------------------+
+   | Labels                   | When to set                                                               |
+   +==========================+===========================================================================+
+   | ``ready-to-merge``       | PR is ready to be merged                                                  |
+   +--------------------------+---------------------------------------------------------------------------+
+
+
+Documentation
+-------------
+
+Building
+~~~~~~~~
 
-4. After enabling the trigger, click again on the wheel.
-5. And click on "Run Trigger Now".
+The documentation has several dependencies which can be installed using pip:
```

Comment:
Worth mention these dependencies already exist in our developer VM

Reply:
added